src/Jenkins/Build.php

```diff
@@ -104,6 +104,41 @@ public function getNumber()
     }
 
     /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return $this->build->description;
+    }
+
+    public function getGitRevision()
```

Comment:
Are these properties specific to git ?
According to their name, it doesn't seem.
Can you please name these functions in a more generic way ?